# Request (empty)
---
# Response
Trace[] traces      # Traces (in sync with above lists)
